### ISSUE TITLE
evaluate DCAT (Data Catalog Vocabulary)

### ISSUE THREAD

**Issue by drewda** [transitland/distributed-mobility-feed-registry#1]
@LeoFrachet has suggested an interesting comparison: the DCAT (Data Catalog Vocabulary) spec, which is used by open data catalogs: https://www.w3.org/TR/vocab-dcat/

Rather than reinvent a new format to describe mobility-specific feeds, why not use this more general purpose format to describe datasets that are connected to each other? It's an interesting question and worth considering.

A particular follow-up question is how DCAT can be us in JSON: https://github.com/ckan/ckanext-dcat/issues/146

**Comment by drewda** [comment]
Potential advantages:

- if a platform like Transitland emits DCAT, users could consume some or all of this and federate in on their own open data portal, like CKAN: https://ckan.org/portfolio/federate/
- if a platform like Transitland consumes DCAT, it could more easily fetch feeds hosted on CKAN, etc.

Potential downsides:
- unclear how to best implement in JSON (vs. heavy RDF/XML)
- would this enable associations between GTFS and GTFS Realtime feeds, MDS and GBFS feeds, etc.?

**Comment by drewda** [comment]
Relevant example: Navitia shares their feed catalog on https://navitia.opendatasoft.com/ (which emits RDF)

**Comment by LeoFrachet** [comment]
When I'll have the time I'll draft a JSON-LD version of DCAT. It should be pretty straightforward, I just have to find the time to do it. Many players are already using such spec, but without clear public specification.

**Comment by drewda** [comment]
Also related: http://vocab.gtfs.org/terms#Feed from https://github.com/OpenTransport/linked-gtfs

**Comment by drewda** [comment]
We're having good luck with DMFR as basic JSON files, along with a JSON Schema. So will close this issue.

In the future, it could be interesting to try also transforming the contents of a DMFR file into DACT, for use by other types of consumers.

**Comment by pietercolpaert** [comment]
> * unclear how to best implement in JSON (vs. heavy RDF/XML)

Using JSON-LD. It’s quite straightforward.

> * would this enable associations between GTFS and GTFS Realtime feeds, MDS and GBFS feeds, etc.?

At least you would be able to filter an opendata-portal on the specification the specific dcat:Distributions comply to. An example of how we described transport datasets in DCAT during a project can be found in this whitepaper by @dachafra: https://oasis.team/storage/app/media/O1.2%20TransportDCAT-AP%20and%20Controlled%20Vocs.pdf

> Also related: http://vocab.gtfs.org/terms#Feed from https://github.com/OpenTransport/linked-gtfs

I created that, but never actually used it in practise for discovery. Open to suggestions on how to extend that though!
